packages/compass-aggregations/src/stores/store.ts

```diff
@@ -252,6 +253,10 @@ const configureStore = (options: ConfigureStoreOptions) => {
     localAppRegistry.on('fields-changed', (fields) => {
       store.dispatch(fieldsChanged(fields.autocompleteFields));
     });
+
+    localAppRegistry.on('open-aggregation-tab', (data) => {
+      store.dispatch(createPipelineFromQuery(data));
+    });
```

Comment:
So I'd suggest either to make the event name more scoped to the use-case you are introducing or event handler should be handling this as a more generic event that can be used to just programmatically open aggregation subtab inside collection tab, optionally indicating somehow that it's coming from query tab AI feature. Currently you are mixing two approaches: the event name is very generic indicating that it can be used to just open aggregation tab, but action that is being dispatched is purely focused on handling this as if it's always tab being opened in relation to AI endpoint returning query instead of aggregation